<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>901</width>
    <height>606</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Телефонный справочник</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>10</y>
      <width>231</width>
      <height>209</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>                                Меню</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="CitizradioButton">
       <property name="text">
        <string>Жители</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="OrgradioButton">
       <property name="text">
        <string>Организации</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_search">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="inputMethodHints">
        <set>Qt::ImhNone</set>
       </property>
       <property name="placeholderText">
        <string>Поиск..</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="searchpushButton">
       <property name="text">
        <string>Искать</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Создать контакт</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="UpdateContactpushButton">
       <property name="text">
        <string>Редактировать выбранный контакт</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="delpushButton">
       <property name="text">
        <string>Удалить выбранный контакт</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>230</y>
      <width>241</width>
      <height>281</height>
     </rect>
    </property>
    <property name="tabletTracking">
     <bool>false</bool>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
    <property name="usesScrollButtons">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Житель</string>
     </attribute>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Имя*</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="nameCitizlineEdit_3">
          <property name="placeholderText">
           <string>Обязательно..</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Фамилия*</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="lastnameCitizlineEdit_4">
          <property name="placeholderText">
           <string>Обязательно..</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Номер*</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="numberCitizrlineEdit_5">
          <property name="placeholderText">
           <string>Обязательно..</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Страна</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="contryCitizlineEdit_6"/>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>Город</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLineEdit" name="cityCitizlineEdit_7"/>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>Адрес</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QLineEdit" name="adresCitizlineEdit_3"/>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>Комментарий</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QLineEdit" name="commentCitizlineEdit_4"/>
        </item>
        <item row="7" column="0">
         <widget class="QPushButton" name="ClearCitizpushButton_3">
          <property name="text">
           <string>Очистить</string>
          </property>
         </widget>
        </item>
        <item row="7" column="1">
         <widget class="QPushButton" name="CreateContactCitizpushButton_2">
          <property name="text">
           <string>Создать</string>
          </property>
         </widget>
        </item>
        <item row="8" column="0">
         <widget class="QPushButton" name="closeTabWPerpushButton_2">
          <property name="text">
           <string>Закрыть</string>
          </property>
         </widget>
        </item>
        <item row="8" column="1">
         <widget class="QPushButton" name="updatePerpushButton">
          <property name="text">
           <string>Обновить контакт</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Организация</string>
     </attribute>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QFormLayout" name="formLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>Название*</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="nameOrglineEdit_9">
          <property name="placeholderText">
           <string>Обязательно..</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_13">
          <property name="text">
           <string>Номер*</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="numberOrglineEdit_10">
          <property name="text">
           <string/>
          </property>
          <property name="placeholderText">
           <string>Обязательно..</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_14">
          <property name="text">
           <string>Страна</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="countryOrglineEdit_11"/>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_15">
          <property name="text">
           <string>Город</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="cityOrglineEdit_12"/>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_16">
          <property name="text">
           <string>Адрес</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLineEdit" name="adresOrglineEdit_3"/>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_17">
          <property name="text">
           <string>Комментарий</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QLineEdit" name="commentOrglineEdit_4"/>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="8" column="0">
         <widget class="QPushButton" name="clearOrgpushButton_2">
          <property name="text">
           <string>Очистить</string>
          </property>
         </widget>
        </item>
        <item row="8" column="1">
         <widget class="QPushButton" name="createOrgpushButton_3">
          <property name="text">
           <string>Создать</string>
          </property>
         </widget>
        </item>
        <item row="9" column="0">
         <widget class="QPushButton" name="closeTabWpushButton_2">
          <property name="text">
           <string>Закрыть</string>
          </property>
         </widget>
        </item>
        <item row="9" column="1">
         <widget class="QPushButton" name="updateOrgpushButton">
          <property name="text">
           <string>Обновить контакт</string>
          </property>
         </widget>
        </item>
        <item row="7" column="1">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>11</y>
      <width>621</width>
      <height>471</height>
     </rect>
    </property>
    <property name="tabletTracking">
     <bool>false</bool>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhHiddenText</set>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::AnyKeyPressed|QAbstractItemView::EditKeyPressed</set>
    </property>
    <property name="sortingEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>490</y>
      <width>211</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QRadioButton" name="setTablePerradioButton">
       <property name="text">
        <string>Контакты жителей</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="setTableOrgradioButton_2">
       <property name="text">
        <string>Контакты организаций</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="num_in_bd">
         <property name="text">
          <string>Количество контактов:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="countPersons">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="exitMainpushButton">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>520</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Выход</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>901</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Главное</string>
    </property>
    <addaction name="action_6"/>
    <addaction name="action_7"/>
    <addaction name="separator"/>
    <addaction name="action_4"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Справка</string>
    </property>
    <addaction name="action"/>
    <addaction name="separator"/>
    <addaction name="action_2"/>
    <addaction name="action_5"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action">
   <property name="text">
    <string>Знакомство</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>О прогремме </string>
   </property>
  </action>
  <action name="action_4">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="action_5">
   <property name="text">
    <string>Версия программы</string>
   </property>
  </action>
  <action name="action_6">
   <property name="text">
    <string>Создать контакт</string>
   </property>
  </action>
  <action name="action_7">
   <property name="text">
    <string>Добавить организацию</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
